tests/conftest.py

```diff
@@ -0,0 +1,12 @@
+"""conftest.py - Common pytest fixtures
+"""
+import tempfile
+from pathlib import Path
+
+import pytest
+
+
+@pytest.fixture
+def tmpdir() -> Path:
```

Comment:
There's an existing fixture that can be used instead and uses pathlib.Path: https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html

Reply:
Right, using it from now on.